CMakeLists.txt

```diff
@@ -18,6 +18,9 @@ set(CMAKE_CXX_FLAGS_RELWITHDEBINFO "-O3 -g")
 set(CMAKE_CXX_FLAGS_RELEASE "-Ofast -fno-finite-math-only")
 set(CMAKE_CXX_FLAGS_DEBUG  "-Wall -Wno-unused-local-typedefs -g")
 
+#Flag to avoid error in compilation
```

Comment:
Please remove these comments if they are no longer needed.

Reply:
Thank you for the changes! Let's clean up the CMakeLists.txt and then I believe it is good to merge.

> I have also added at /examples/test_fm.cpp the library due to the error I commented before.

What do you mean there? I do not see any changes on that file.

Saludos a vosotros también! :D